RELEASE_NOTES.md

```diff
@@ -0,0 +1,70 @@
+
+## Intel Architecture Processors
+* Introduced support for `f32` convolution with `fp16` compressed weights.
+* Enabled support for int8 or int4 compressed weights in matmul primitive. This functionality is implemented on Intel CPUs.
+
+## Generic SYCL backend
+* Introduced support for RNN Vanilla primitive forward propagation.
+* Introduced support for inner product primitive backward propagation.
+* Introduced support for group normalization primitive.
+* Improved precision of inner product primitive with sum post-op for larger shapes.
+
+## AArch64-based Processors
+* Enabled FP16 for JIT reorder kernels.
+* Enabled matmul static quantization.
+
+## NVIDIA GPUs
+* Introduces Graph API support.
+
+# Usability
+ 
+* Added support for group normalization primitive with [ONEDNN_ENABLE_PRIMITIVE](https://uxlfoundation.github.io/oneDNN/dev_guide_build_options.html#onednn-enable-primitive) build option.
+* Enabled ROCm 6 support for AMD GPUs.
+* Improved CMake integration when installing oneDNN with the Nvidia backend enabled.
+
+## AArch64-based Processors
+ * Default the number of threads to max for acl_threadpool. Fixes a crash in Tensorflow.
+ * Fixed scratchpad being ignored for some GEMMs. Reduces memory and speeds up execution.
+ * Fixed a bug in FP32 reorders where ACL would return incorrect results.
+
+# Validation
+* Added benchdnn option [`--execution-mode`](https://github.com/uxlfoundation/oneDNN/blob/rls-v3.8/tests/benchdnn/doc/knobs_common.md#--execution-mode) to test oneDNN functionality with SYCL Graph record/execute mode.
+* Extended benchdnn option [`--cold-cache`](https://github.com/uxlfoundation/oneDNN/blob/main/tests/benchdnn/doc/knob_cold_cache.md) with support for cold TLB mode.
+* Added benchdnn option `--bia-dt` to control bias data type for matmul, inner product, convolution, and deconvolution.
+* Extended syntax of benchdnn `--dt` option in [Graph API driver](https://github.com/uxlfoundation/oneDNN/blob/main/tests/benchdnn/doc/driver_graph.md) to manage data types of individual tensors in a pattern.
+
+# Deprecated Functionality
+
+# Breaking Changes
+* Removed experimental [Graph Compiler](https://uxlfoundation.github.io/oneDNN/v3.7/dev_guide_graph_compiler.html) backend.
+
+# Thanks to these Contributors
+This release contains contributions from the project core team as well as Alexander Simonov @asimonov1, Denis @redradist, Dmitriy Ovchinnikov @inteldimitrius, Eliezer Weissmann @eliezerweissmann, Hubert Maciak @hmaciak, Ilya Lavrenov @ilya-lavrenov, James McGregor @Jmc18134, Marek Michalowski @michalowski-arm, Maria Zhukova @mzhukova, Orel Yehuda @yehudaorel, Ravi Pushkar @rpushkarr, Renato Barros Arantes @renato-arantes, Shreyas-fuj @Shreyas-fuj, Shu Chen @shu1chen, Viktoriia Gvozdeva @vgvozdeva, Yair Obodovsky @yair-obodovsky, hmaciak @hmaciak, jstachowintel @jstachowintel, zhangfei @zhangfeiv0, James McGregor @Jmc18134, Marek Michalowski @michalowski-arm, Renato Barros Arantes @renato-arantes.
```

Comment:
I would leave the usernames only, or put them in the brackets to improve readability.